dts/bindings/clock/nordic,nrf-lfclk.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: nRF LFCLK (Low Frequency CLocK)
+
+compatible: "nordic,nrf-lfclk"
+
+include: fixed-clock.yaml
+
+properties:
+  clock-frequency:
+    const: 32768
+
+  lfrc-accuracy-ppm:
+    type: int
+    description: Clock accuracy in parts per million if LFRC clock source is used.
+
+  lflprc-accuracy-ppm:
+    type: int
+    description: Clock accuracy in parts per million if LFLPRC clock source is used.
+
+  hfxo-clock:
+    type: phandle
+    description: |
+      The HFXO clock is used as a clock source if the SYNTH clock source is selected
+      and the LFXO clock is not available.
+
+  lfxo-clock:
+    type: phandle
+    description: |
+      The LFXO clock is used as a clock source if the SYNTH clock source is selected
+      and the LFXO clock is available.
```

Comment:
You could add a note to both that these clocks are used indirectly through FLL16M in BYPASS mode. Something like:

The xxXO clock is used indirectly as a clock source, through FLL16M in the BYPASS mode, if the SYNTH clock source is selected...

Or maybe easier to understand:

The xxXO clock is used as a clock source if...
The xxXO is used indirectly. It is a clock source for FLL16M configured in a BYPASS mode which is a direct clock source for LFCLK in the SYNTH mode.

Reply:
updated to mention indirect use through FLL16M